website/cue/reference/components/sources/socket.cue

```diff
@@ -154,15 +154,18 @@ components: sources: socket: {
 	]
 
 	telemetry: metrics: {
-		events_in_total:                  components.sources.internal_metrics.output.metrics.events_in_total
-		connection_errors_total:          components.sources.internal_metrics.output.metrics.connection_errors_total
-		connection_failed_total:          components.sources.internal_metrics.output.metrics.connection_failed_total
-		connection_established_total:     components.sources.internal_metrics.output.metrics.connection_established_total
-		connection_failed_total:          components.sources.internal_metrics.output.metrics.connection_failed_total
-		connection_send_errors_total:     components.sources.internal_metrics.output.metrics.connection_send_errors_total
-		connection_send_ack_errors_total: components.sources.internal_metrics.output.metrics.connection_send_ack_errors_total
-		connection_shutdown_total:        components.sources.internal_metrics.output.metrics.connection_shutdown_total
-		component_received_bytes_total:   components.sources.internal_metrics.output.metrics.component_received_bytes_total
-		component_received_events_total:  components.sources.internal_metrics.output.metrics.component_received_events_total
+		events_in_total:                      components.sources.internal_metrics.output.metrics.events_in_total
+		connection_errors_total:              components.sources.internal_metrics.output.metrics.connection_errors_total
+		connection_failed_total:              components.sources.internal_metrics.output.metrics.connection_failed_total
+		connection_established_total:         components.sources.internal_metrics.output.metrics.connection_established_total
+		connection_failed_total:              components.sources.internal_metrics.output.metrics.connection_failed_total
+		connection_send_errors_total:         components.sources.internal_metrics.output.metrics.connection_send_errors_total
+		connection_send_ack_errors_total:     components.sources.internal_metrics.output.metrics.connection_send_ack_errors_total
+		connection_shutdown_total:            components.sources.internal_metrics.output.metrics.connection_shutdown_total
+		component_discarded_events_total:     components.sources.internal_metrics.output.metrics.component_discarded_events_total
+		component_errors_total:               components.sources.internal_metrics.output.metrics.component_errors_total
+		component_received_bytes_total:       components.sources.internal_metrics.output.metrics.component_received_bytes_total
```

Comment:
Is `BytesReceived` emitted for every `mode` already? I see it was already documented as part of this source's metrics and see it [here for TCP](https://cs.github.com/vectordotdev/vector/blob/4629f4ad8c5e9d124f29249835b3c7a0592f4f25/src/sources/util/tcp/mod.rs#L266), but just checking for the others.

Reply:
It was just emitted for TCP socket but I'm adding UDP 😉